CONTRIBUTING.md

```diff
@@ -61,7 +61,7 @@ The core implementation of this charm is represented by the [`GrafanaAgentOperat
 
 - `self.on.install`: In this event we patch K8s service ports.
```

Comment:
The PR description is rather terse : "Clean up the implementation of Prometheus Remote Write in Grafana Agent." . It is not clear why the clean up was required and what were the cleanups that were required.

Reply:
The most important reason was that alerts were not forwarded to prometheus